### PR TITLE
Allow to override cookie path

### PR DESCRIPTION
**Pull request by ivankristianto**
Fix: #9 

This PR add an optional param to `consent_api_set_cookie` for the path. Allowing the application to override this in multisite subdirectory mode.

### CODE REVIEW

File: src/wp-consent-api.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function wp_has_consent(category) {
  * @param {string} name The cookie name to set.
  * @param {string} value The cookie value to set.
```
Comment (jazzsequence) [inline src/wp-consent-api.js:54]: Let's make sure there's a docblock parameter for the new variable and then this is good to go.

**Review by jazzsequence — CHANGES_REQUESTED** [review]
Add some doc comments for the new param and we're good.

This is probably something we could also copy upstream to https://github.com/rlankhorst/wp-consent-level-api -- would you like to create a PR for that or do you want me to? (I'm fine either way.)

@roborourke wdyt of including this in v6?

**Comment by roborourke** [comment]
Happy for minor changes like this to be included in v6, I'll leave it up to you to decide whether this is done as a minor or patch release but I guess it might warrant an update to the privacy module version constraint if you do a minor release - totally fine to do.

File: src/wp-consent-api.js (hunk 1 above)
Comment (ivankristianto) [reply src/wp-consent-api.js:54]: Ahh, forgive my rush. I did add the additional param to doc block in this commit: https://github.com/humanmade/consent-api-js/pull/10/commits/779b488e3c8173e3f2da140a1ff397d58750412c

Comment (ivankristianto) [reply src/wp-consent-api.js:54]: > This is probably something we could also copy upstream to https://github.com/rlankhorst/wp-consent-level-api -- would you like to create a PR for that or do you want me to? (I'm fine either way.)

I'm happy to do this.

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**